### PR TITLE
Clarify documentation of EnvironmentPlugin

### PR DESCRIPTION
**Pull request by simon04**
Follow-up of #620.

### CODE REVIEW

File: content/plugins/environment-plugin.md [hunk 1]
```diff
@@ -6,63 +6,74 @@ contributors:
...
+The EnvironmentPlugin accepts either an array of keys.
 
-```js
-new webpack.EnvironmentPlugin(['NODE_ENV', 'VERSION'])
+```javascript
+new webpack.EnvironmentPlugin(['NODE_ENV', 'DEBUG'])
 ```
 
-When given an object, the values are used as defaults for when they are not defined in `process.env`.
+This is equivalent to the following DefinePlugin application:
```
Comment (bebraw) [inline content/plugins/environment-plugin.md]: `DefinePlugin` over DefinePlugin. You could do the same change for the rest as it reads a little better then.

Comment (simon04) [inline content/plugins/environment-plugin.md]: We are not very consistent on that in other guides:
* https://webpack.js.org/plugins/ – lowercase w/o ticks
* http://localhost:3000/plugins/loader-options-plugin – lowercase w/ ticks

Shall we go for `CamelCase` for all plugins?

Comment (bebraw) [inline content/plugins/environment-plugin.md]: Yeah, that's a good addition to the writer's guide too. I think it's a good idea as then you have a bit of meta in the text (plugins pop out).

Comment (simon04) [inline content/plugins/environment-plugin.md]: I'll do a separate PR then.

Comment (bebraw) [inline content/plugins/environment-plugin.md]: Ok. Poke me when you want me to look into something. Just adding as a reviewer seems to do the trick.

Comment (simon04) [inline content/plugins/environment-plugin.md]: Can we merge this PR first to avoid merge conflicts?

Comment (bebraw) [inline content/plugins/environment-plugin.md]: Yup.

Comment (simon04) [inline content/plugins/environment-plugin.md]: Will you?

Comment (bebraw) [inline content/plugins/environment-plugin.md]: Done. Sorry for the delay (dinner time).

**Pull request merged**